jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -199,7 +200,11 @@ boolean doCommand(InputStream cmdStream, OutputStream respStream) {
 							+ " " + attachError); //$NON-NLS-1$
 				}
 			} else if (cmd.startsWith(Command.GET_SYSTEM_PROPERTIES)) {
-				replyWithProperties(com.ibm.oti.vm.VM.getVMLangAccess().internalGetProperties());
+				Properties internalProperties = com.ibm.oti.vm.VM.getVMLangAccess().internalGetProperties();
+				String argumentString = String.join(" ", com.ibm.oti.vm.VM.getVMArgs()); //$NON-NLS-1$
+				Properties newProperties = (Properties) internalProperties.clone();
```

Comment:
Rather than `clone` you should use `new Properties(internalProperties)`.

Reply:
@keithc-ca I tried that, but `new Properties(internalProperties)` sets `internalProperties` as "defaults", so they don't show up as first-class properties in `Properties.store()`, i.e.
```
jshell> Properties P = System.getProperties()
P ==> {awt.toolkit=sun.lwawt.macosx.LWCToolkit, java.sp ... , java.class.version=55.0}

jshell> Properties p2 = new Properties(P)
p2 ==> {}

jshell> p2.getProperty("java.class.version")
$5 ==> "55.0"

jshell> P.store(System.out,"");
#
#Wed Feb 20 11:33:42 EST 2019
awt.toolkit=sun.lwawt.macosx.LWCToolkit
java.specification.version=11
file.encoding.pkg=sun.io

jshell> p2.store(System.out,"");
#
#Wed Feb 20 11:33:57 EST 2019

```

Reply:
That's disappointing.